python/tritonserver/_api/_response.py

```diff
@@ -360,13 +104,12 @@ class InferenceResponse:
     @staticmethod
     def _from_tritonserver_inference_response(
         model: _model.Model,
-        request: TRITONSERVER_InferenceRequest,
         response,
         flags: TRITONSERVER_ResponseCompleteFlag,
+        output_memory_type: Optional[DeviceOrMemoryType] = None,
```

Comment:
maybe instead of passing output_memory_type alone - we could pass the entire original request object and also use it to set the response id - incase the response is none?

Reply:
yes, updated: [Update inference response conversion interface](https://github.com/triton-inference-server/core/pull/426/commits/023a1459d84d26b42a80b659de2a5882cb62a4f8)